apps/backend/strava.py

```
@@ -144,55 +146,94 @@ async def refresh_access_token(force: bool = False) -> str | None:
+                if is_retryable and attempt < _REFRESH_MAX_ATTEMPTS:
+                    logger.warning(
+                        f"{last_error_msg} - retrying in {backoff_seconds:.1f}s "
+                        f"(attempt {attempt}/{_REFRESH_MAX_ATTEMPTS})"
+                    )
+                    await asyncio.sleep(backoff_seconds)
+                    backoff_seconds *= 2
+                    continue
+
+                logger.error(last_error_msg)
+                break
+            except (httpx.RequestError, asyncio.TimeoutError) as e:
```

Ruff (0.15.11)

[warning] Replace aliased errors with `TimeoutError`

Replace with builtin `TimeoutError`

(UP041)